dist/react-simple-currency.js

```diff
@@ -50,7 +50,7 @@ var SimpleCurrencyInput = (function (_React$Component) {
       if (nextProps.value || nextProps.value === 0) {
         this.setState({
           rawValue: nextProps.value,
-          isNegative: nextProps.allowNegative && this.includesNegativeSymbol(nextProps.value)
+          isNegative: nextProps.allowNegative && (this.includesNegativeSymbol(nextProps.value) || this.state.isNegative && nextProps.value === 0)
```

Comment:
can we add another set of parentheses to identify which set of conditions are supposed to be executed together? from some changes below I assume you want:

<pre>
(this.includesNegativeSymbol(nextProps.value) || (this.state.isNegative && nextProps.value === 0))
</pre>

would be helpful to indent the same as you have in src/SimpleCurrencyInput.js